content/reference/version-compatibility/index.md

```diff
@@ -43,7 +43,7 @@ The following lists our Terraform packages and their compatibility with Terrafor
 | [terraform-aws-consul](https://github.com/hashicorp/terraform-aws-consul)                        | <=v0.6.1         | >=v0.7.0         |
 | [terraform-aws-vault](https://github.com/hashicorp/terraform-aws-vault)                          | <=v0.12.2        | >=v0.13.0        |
 | [terraform-kubernetes-helm](https://github.com/gruntwork-io/terraform-kubernetes-helm)           | <=v0.4.0         | >=v0.5.0         |
-
+| [terraform-aws-nomad](https://github.com/hashicorp/terraform-aws-nomad)                          | <=v0.4.5         | >=v0.5.0         |
```

Comment:
Can you put this above `terraform-kubernetes-helm`, so that it is with the other hashicorp repos?

Reply:
done.